CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-vc-issuer ChangeLog
 
+## 0.1.1 -
```

Comment:
This package hasn't been released yet.  0.1.0 below is the first version (or renamed as 1.0.0 if that's more appropriate).  Doesn't make much sense to add just one specific entry for the first version that has 100+ commits.

Reply:
change it to `1.0.0` as the version in the real notes?

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-vc-issuer/pull/27/commits/95fb1ef9f23bb5a0a93ca1d144fe9929c217961b